wordpress/wp-content/plugins/cds-base/classes/Modules/Notify/ListManagerUserProfile.php

```diff
@@ -58,9 +79,14 @@ public function updateListManagerMeta($userId)
 
         $user = new WP_User($userId);
         $user->remove_cap('list_manager_bulk_send');
+        $user->remove_cap('list_manager_bulk_send_send');
```

Comment:
This is probably `list_manager_bulk_send_sms`, not `list_manager_bulk_send_send`